_topic_maps/_topic_map.yml

```diff
@@ -3354,13 +3354,15 @@ Topics:
       File: oadp-using-data-mover-for-csi-snapshots-doc
     - Name: Using OADP 1.2 Data Mover with Ceph storage
       File: oadp-12-data-mover-ceph-doc
-  - Name: OADP 1.3 Data Mover
+  - Name: OADP 1.3 Data Mover and later
     Dir: installing
     Topics:
     - Name: About the OADP 1.3 Data Mover
       File: about-oadp-1-3-data-mover
     - Name: Backing up and restoring volumes by using CSI snapshots data movement
       File: oadp-backup-restore-csi-snapshots
+    - Name: Overriding Kopia hashing, encryption, and splitter algorithms
```

Comment:
Consider "Overriding algorithms" unless the three listed are not the complete list?

Reply:
Done.